packages/frontend/src/hooks/useEcharts.ts

```diff
@@ -338,6 +338,12 @@ const getEchartAxis = ({
                       })
                     : xAxisConfiguration?.[0]?.name ||
                       (xAxisItem ? getItemLabel(xAxisItem) : undefined),
+                min:
+                    validCartesianConfig?.eChartsConfig?.xAxis?.[0]?.min ||
```

Comment:
I believe we need to use `yAxisConfiguration` and `xAxisConfiguration` variables so it works when the axis are flipped.